fretboard.el

```diff
@@ -31,20 +32,20 @@
 ;; To highlight semitones, prefix the key with # (yes, prefix!).
 
 ;;; Code:
+(require 'symbol-overlay)
 
 (defgroup fretboard nil
-  "A guitar fretboard visualization.")
+  "A guitar fretboard visualizer."
+  :group 'fretboard)
 
 (defgroup fretboard-faces nil
   "Faces in `fretboard-mode'."
   :group 'fretboard)
 
 (defun fretboard-highlight-note ()
   "Highlight the note corresponding to the pressed key.
-
 The key is being read directly from the used keybinding."
   (interactive)
-  (require 'symbol-overlay)
```

Comment:
My intention was to load it only when using the highlighting functionality, but you're probably right it's overengineered.

Reply:
The byte compiler does will not load the dependency during byte compilation otherwise.

Reply:
Yes, I'm aware. I meant that this little added amount of lazy loading is not worth having byte compilation warnings. Or complicating the code with `eval-when-compile`.